app/public/lobby/lobby.js

```diff
@@ -0,0 +1,49 @@
+// temp: lobby id won't be hard coded to 1 all the time, it will be
+// dynamically assigned once full lobby system is ready
+const LOBBY_ID = 1;
+
+// TO DO: This username wont be randomly generated each time
+// ideally, we use a cookie to keep track of the user rather than a hard coded username to persist between refreshes
+// in the backend, a cookie already gets sent to the client when we hit /lobby/join to set the username
+// we just need to hook up logic to use that
+// also it'd be nice to have this implemented as a on/off toggle development purposes so we could open two tabs to simulate two different players
+// or toggle it off and use the cookie to simulate one player logging out then rejoining the game
+const username = Math.floor(Math.random()*10000);
```

Comment:
Yeah this is temporary, the cookie stuff isn't set up yet since this was already a huge ticket but if you look I do set a cookie using the username in the backend. Once the login stuff is done I'll refactor to use cookie for anonymous users and logged in users